packages/common/src/Widget.ts

```diff
@@ -66,10 +67,13 @@ export abstract class Widget extends PropertyExt {
 
     protected _overlayElement;
 
+    protected _logger;
+
     constructor() {
         super();
         this._class = Object.getPrototypeOf(this)._class;
         this._id = this._idSeed + widgetID++;
+        this._logger = scopedLogger(this._id);
```

Comment:
Interesting - I only envisioned using the scoping for "per file" not "per widget instance"